app/services/encryption/multi_region_kms_client.rb

```diff
@@ -0,0 +1,78 @@
+require 'json'
+module Encryption
+  class MultiRegionKMSClient
```

Comment:
Is there a corresponding spec file for this class? I really like having a unit test that corresponds directly to a class like this, because it means that when making changes to a class, it's really clear where the corresponding specs should go